EXPORT_HELPERS.md

```diff
@@ -0,0 +1,215 @@
+## Proposal / Discussion Draft: Consistent & Documented Export Formats
+
+Goal of this proposal is to produce consistent and conformant formatted exports of the VSS via
+the VSS-TOOLS or other potential 3rd-party implementations, resulting in a high degree of stability and
+interoperability between implementations using a particular format (such as IDL).
+
+This consistency and stability can ease the creation of interoperable examples and reference implementations for a given format.  
+This draft focuses on IDL, but is intended to also maximize interoperability within other export formats.
+
+To enable this capability, the following changes are proposed:
+
+- Create a document to describe the recommended formatting and structure of the resulting exported (IDL or other) files.
+- Modify the exporters in VSS-TOOLS to conform to the above document.
+- Utilize the `arraysize` and `stringsizemax` to set the dimensions of the exported array/sequence/string types.
+- Introduce 3 new keywords to the VSS: `memberof`, `membergroup`, `isogroup`, used as:

```

Comment:
I can see the value of this, for some exporters, but for others it woudl be of no relevance (CSV, JSON), the question is, should this be defined as "Core VSS", or maybe as reserved atrributes in an overlay (and we could still think about shipping/providing one for standard catalogue).

The challenge I see is, if we make the "core" VSS to powerful, basically every exporter/techstack, would feel like "I am VSS compliant except x,y,z is not supported"